### DISCUSSION TITLE
basic/computer_architecture_principles/

### DISCUSSION THREAD

**Discussion by giscus** [luguosong/programming-notes#207] [Announcements]
# basic/computer_architecture_principles/

http://127.0.0.1:8000/basic/computer_architecture_principles/

**Comment by luguosong** [comment]
教材：[计算机组成与设计](https://book.douban.com/subject/35998323/)

![](https://cdn.jsdelivr.net/gh/luguosong/images@master/blog-img/202404271937516.jpeg)
